src/types/formatted/healthCareLocalFacility.ts

```diff
@@ -43,5 +44,101 @@ export type HealthCareLocalFacility = PublishedEntity & {
   socialLinks: FacilitySocialLinksProps
   lovellVariant?: LovellChildVariant
   lovellSwitchPath?: string
-  healthServices: NodeHealthCareLocalHealthService[] // TODO: Format these
+  healthServices: FormattedVAMCFacilityHealthServices[]
+}
+
+/** Represents the "single" object containing service-related fields. */
+interface VamcFacilityServiceLocation {
+  /** Type of office visits supported (e.g., "no", "yes_appointment_only"). */
+  fieldOfficeVisits?: string
+  /** Type of virtual support available (e.g., "yes_veterans_can_call"). */
+  fieldVirtualSupport?: string
+  /** Appointment intro text type ("remove_text", "customize_text", "use_default_text"). */
+  fieldApptIntroTextType?: 'remove_text' | 'customize_text' | 'use_default_text'
+  /** Custom appointment intro text (if applicable). */
+  fieldApptIntroTextCustom?: string
+  /** Array of additional phone numbers for appointments or contact. */
+  fieldOtherPhoneNumbers?: PhoneNumber[]
+  /** Indicates if online scheduling is available ("yes" or others). */
+  fieldOnlineSchedulingAvail?: 'yes' | string
+  /** Array of additional contact phone numbers. */
+  fieldPhone?: PhoneNumber[]
+  /** Array of email contact objects. */
+  fieldEmailContacts?: EmailContact[]
+  /** Service hours configuration ("0" for facility hours, "1" for unspecified, "2" for specific hours). */
+  fieldHours?: '0' | '1' | '2' | string
+  /** Specific service office hours (array of hour objects). */
+  fieldOfficeHours?: FieldOfficeHours[]
+  /** Additional information about service hours. */
+  fieldAdditionalHoursInfo?: string
+  /** Indicates if the main facility phone number should be used (true/false). */
+  fieldUseMainFacilityPhone?: boolean
+  /** Indicates if the facility phone number should be used for appointments (true/false). */
+  fieldUseFacilityPhoneNumber?: boolean
+  /** Optional service location address entity. */
+  fieldServiceLocationAddress?: ParagraphServiceLocationAddress
+}
+
+/** Represents the main input data structure for the service location template. */
+export interface ServiceLocationTemplateData {
+  /**
+   * Indicates if a referral is required
+   * "0" for no
+   * "1" for yes
+   * "2", "not_applicable", or "unknown" to omit the referral section
+   */
+  fieldReferralRequired?:
+    | '0'
+    | '1'
+    | '2'
+    | 'not_applicable'
+    | 'unknown'
+    | string
```

Comment:
If we allow `string` here do we need any of the others?

Reply:
TL;DR: Without `| string`, TS gets angry, but we don't expect it to be anything other than the listed string literals.

So the way I typically use this (and should probably document in a comment) is by making the string literals in the union the _expected_ options, but because we're getting data from Drupal, we have no static type guarantee that the string we receive is _absolutely_ going to be one of these. Once we get runtime data validation through the likes of Zod, we'll be able to guarantee that the strings are one of these options.

That said, since we _are_ doing `as Whatever` type assertions when we grab the data from either the mock data or the query to Drupal, this is maybe unnecessary. 🤷 🙈